cvat-cli/src/cvat_cli/_internal/common.py

```diff
@@ -102,3 +113,77 @@ def build_client(parsed_args: argparse.Namespace, logger: logging.Logger) -> Cli
     client.organization_slug = popattr(parsed_args, "organization")
 
     return client
+
+
+def configure_function_implementation_arguments(parser: argparse.ArgumentParser) -> None:
+    function_group = parser.add_mutually_exclusive_group(required=True)
+
+    function_group.add_argument(
+        "--function-module",
+        metavar="MODULE",
+        help="qualified name of a module to use as the function",
+    )
+
+    function_group.add_argument(
+        "--function-file",
+        metavar="PATH",
+        type=Path,
+        help="path to a Python source file to use as the function",
+    )
+
+    parser.add_argument(
+        "--function-parameter",
+        "-p",
+        metavar="NAME=TYPE:VALUE",
+        type=parse_function_parameter,
+        action=BuildDictAction,
+        dest="function_parameters",
+        help="parameter for the function",
+    )
+
+    original_executor = parser.get_default("_executor")
```

Comment:
Consider adding a named constant for this string.

Reply:
I'm not sure if that's needed here. It is basically a secret command-line argument, and we don't usually create constants for those.